lib/utils.js

```
@@ -647,80 +697,84 @@ function splitMsg(msg, encoding) {
 	} else if (defs.constsById.MESSAGE_STATE[status]) {
-		status	= parseInt(status);
+		status = parseInt(status);
 	} else {
-		status	= 5; // UNDELIVERABLE
+		status = 2; // UNDELIVERABLE
 	}
 
-	if (typeof cb !== 'function') {
-		cb = function () {};
+	if (typeof callback !== 'function') {
+		callback = function() {};
```

ESLint

[error] Missing space before function parentheses.

(space-before-function-paren)